Classes/common/CDTDatastore.m

```diff
@@ -150,6 +168,11 @@ - (NSUInteger)documentCount
 
 - (NSString *)name { return self.database.name; }
 
+- (id<CDTEncryptionKeyProvider>)copyEncryptionKeyProvider
+{
+    return [_encryptionKeyProvider copy];
+}
```

Comment:
Could this just be `encryptionKeyProvider`? Does the caller need to know it's a copy?